cli/util/uploader.go

```diff
@@ -40,7 +40,7 @@ const (
 // Constants related to retry mechanism.
 const (
 	// Maximum number of retries.
-	maxRetries = 5
+	maxRetries = 100
```

Comment:
It's only about half an hour of retries.

Reply:
plus requests timeouts

Reply:
Have you tried to upload index files to mainnet with `10`?